conf/v0.6/general/test_scenario/chakra_replay.toml

```diff
@@ -1,6 +1,6 @@
 name = "chakra_replay"
 
-[Tests]
-  [Tests.1]
-    name = "chakra_replay"
-    num_nodes = "2"
+[[Tests]]
+test = "chakra_replay"
+name = "Test.1"
```

Comment:
How about switching the two lines so that the name field appears before the test field? Let's see what @srinivas212 thinks before we make changes.